### PR TITLE
Pp 142

### PR DESCRIPTION
**Pull request by acceleanu**
JIRA [https://payments-platform.atlassian.net/browse/PP-142]

### CODE REVIEW

**Comment by acceleanu** [comment]
rebuild

File: src/main/java/uk/gov/pay/connector/service/smartpay/SmartpayCaptureResponse.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    private String captureResponse;
+
+    @XmlPath("soap:Body/soap:Fault/faultstring/text()")
+    private String errorMessage;
+
+    public boolean isCaptured() {
+        return isNotBlank(captureTransactionId) && "[capture-received]".equalsIgnoreCase(captureResponse);
+    }
+
+    public String getPspRefrence() {
```
Comment (mikemey) [inline src/main/java/uk/gov/pay/connector/service/smartpay/SmartpayCaptureResponse.java]: typo in method name

File: src/test/java/uk/gov/pay/connector/it/dao/ChargeDaoITest.java [hunk 2]
```diff
@@ -7,6 +7,7 @@
 import org.junit.rules.ExpectedException;
 import uk.gov.pay.connector.dao.ChargeDao;
 import uk.gov.pay.connector.dao.PayDBIException;
+import uk.gov.pay.connector.model.api.ExternalChargeStatus;
```
Comment (mikemey) [inline src/test/java/uk/gov/pay/connector/it/dao/ChargeDaoITest.java]: unused import, please remove.

File: src/test/java/uk/gov/pay/connector/it/resources/worldpay/WorldpayCardResourceITest.java [hunk 3]
```diff
@@ -2,6 +2,13 @@
 
 import org.junit.Test;
 import uk.gov.pay.connector.it.base.CardResourceITestBase;
+import uk.gov.pay.connector.model.api.ExternalChargeStatus;
+import uk.gov.pay.connector.model.domain.ChargeStatus;
```
Comment (mikemey) [inline src/test/java/uk/gov/pay/connector/it/resources/worldpay/WorldpayCardResourceITest.java]: both imports on line 5+6 unused, please remove.

File: src/main/java/uk/gov/pay/connector/service/smartpay/SmartpayCaptureResponse.java (hunk 1 above)
Comment (rahul-vinayak) [inline src/main/java/uk/gov/pay/connector/service/smartpay/SmartpayCaptureResponse.java]: too many e's in there, fixed anyway

File: src/test/java/uk/gov/pay/connector/it/resources/worldpay/WorldpayCardResourceITest.java (hunk 3 above)
Comment (rahul-vinayak) [inline src/test/java/uk/gov/pay/connector/it/resources/worldpay/WorldpayCardResourceITest.java]: :+1:

**Comment by bazbremner** [comment]
Would I have a summary of the purpose of this - and other - PRs so it's clear what the state of the code is, please?

Looking at PRs as a newbie to the team (and I can imagine going back and reviewing PRs and changes in future) is much easier with a little bit of context.
I can also set my expectations appropriately if I know if a PR is a spike, first cut, mostly complete or feature complete and what things you thing are worthy of comment.

**Comment by mikemey** [comment]
:+1:

**Pull request merged**